public/samples/interop-samples/interop-examples.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<html>
+<head>
+  <title>Fable Interop Examples by Gary Bilkus</title>
+  <meta http-equiv='Content-Type' content='text/html; charset=utf-8'>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" href="fable.ico" />
+  <script src="https://cdn.polyfill.io/v2/polyfill.js?features=es6,fetch"></script>
+</head>
+<body>
+  <!-- The script below allows us to display the result of console logs to a div -->
+  <style>
+      .console-log-div {
+        border: 1px solid gray;
+        padding: 5px 10px;
+        border-radius: 5px;
+        width: 90% !important;
+        height: 300px !important;
+        background-color: #efefef;
+        overflow-y: scroll;
+      }
+  </style>
+  <script src="https://rawgit.com/bahmutov/console-log-div/master/console-log-div.js"></script>
```

Comment:
I would write the explanation text here and with something like:

```fs
  <h2>
    The section below contains the console output from running the program.
  </h2>
  <h4>What's most interesting is the generated code, so switch to that tab and compare with the original program.
  </h4>
  <p>Version 1.0 March 2018</p>
```

It's less agressive :)